packages/vuetify/src/components/VOverlay/VOverlay.tsx

```diff
@@ -93,7 +93,10 @@ export const makeVOverlayProps = propsFactory({
   disabled: Boolean,
   opacity: [Number, String],
   noClickAnimation: Boolean,
-  modelValue: Boolean,
+  modelValue: {
+    type: Boolean as PropType<boolean | null>,
+    default: null,
+  },
```

Comment:
Why?

Reply:
Oops things got a bit out of hand there. This cast should be removed.